src/electionguard_gui/services/decryption_stages/decryption_s1_join_service.py

```diff
@@ -0,0 +1,45 @@
+from pymongo.database import Database
+from electionguard.ballot import BallotBoxState
+
+from electionguard_gui.models.decryption_dto import DecryptionDto
+from electionguard_gui.services.decryption_stages.decryption_stage_base import (
+    DecryptionStageBase,
+    get_tally,
+)
+
+
+class DecryptionS1JoinService(DecryptionStageBase):
+    """Responsible for the 1st stage during a decryption were guardians join the decryption"""
+
+    def run(self, db: Database, decryption: DecryptionDto) -> None:
+        current_user_id = self._auth_service.get_required_user_id()
+        self._log.info(f"S1: {current_user_id} decrypting  {decryption.decryption_id}")
+        election = self._election_service.get(db, decryption.election_id)
+        manifest = election.get_manifest()
+        context = election.get_context()
+
+        guardian = self._guardian_service.load_guardian_from_decryption(
+            current_user_id, decryption
+        )
+        ballots = self._ballot_upload_service.get_ballots(db, election.id)
+        spoiled_ballots = [
+            ballot for ballot in ballots if ballot.state == BallotBoxState.SPOILED
+        ]
+        ciphertext_tally = get_tally(manifest, context, ballots)
```

Comment:
Just checking that the get_tally call can handle the ballots that are cast AND spoiled.

Reply:
Thanks for double checking.  I had to think about it, but this is correct as is.  The `ballots` variable will contain all ballots both cast AND spoiled.